src/translations/fr/subscribe/confirm.mdx

```diff
@@ -0,0 +1,7 @@
+# Presque terminé!
+
+**Vous devriez recevoir un e-mail avec un lien de confirmation.**
```

Comment:
Please use consistent syntax in the whole text: either “e-mail” or “email” (or, in French, “courriel” would be welcome  🙂).